src/common/utils/recommend.ts

```diff
@@ -79,11 +79,29 @@ export function getRecommendations(
     item => item.category === RecommendCategory.GATHERINGS,
   );
 
-  const [masksRecommendation, otherRecommendations] = partition(
+  const [masksRecommendation, recommendationsWithoutMasks] = partition(
     recommendationsWithoutGatherings,
     item => item.category === RecommendCategory.MASKS,
   );
 
+  const [
+    vaccinationRecommendation,
+    recommendationsWithoutVaccination,
+  ] = partition(
+    recommendationsWithoutMasks,
+    item => item.category === RecommendCategory.VACCINATION,
+  );
+
+  const [boosterRecommendation, recommendationsWithoutBooster] = partition(
+    recommendationsWithoutVaccination,
+    item => item.category === RecommendCategory.BOOSTER,
+  );
+
+  const [testingRecommendation, otherRecommendations] = partition(
+    recommendationsWithoutBooster,
+    item => item.category === RecommendCategory.TESTING,
+  );
```

Comment:
This is fine since it's following the existing pattern, but it's getting pretty repetitive, which makes it harder to read and more likely for somebody to make mistakes when changing it.  I think we could restructure this to be easier to read / more maintainable by turning this into a for loop something like:

```
const recommendationCategoryOrdering = [
  RecommendCategory.MASKS,
  RecommendCategory.GATHERINGS,
  ...
];

const remainingRecommendations = recommendations;
const orderedRecommendations: Recommendation[] = [];
for (const category of recommendationCategoryOrdering) {
  const [inCategory, notInCategory] = partition(remainingRecommendations, item => item.category === category);
  orderedRecommendations = orderedRecommendations.concat(inCategory);
  remainingRecommendations = notInCategory;
}
```

That said, if you don't want to refactor this code, that's fine.  Maybe add a:

```
// TODO(fai): Consider refactoring this repetitive partition() code into a for loop or similar.
```

If you don't feel comfortable with your name on it, feel free to do TODO(michael). 😁

Reply:
I agree. I was thinking it's getting repetitive as I was adding three more categories into this 😆 . I implemented what you suggested.